src/GeositeFramework/css/app.css

```diff
@@ -262,8 +262,8 @@ input.code-like, textarea.code-like {
             }
         .content .map .tools {
             position: absolute;
-            top: 92px;
-            left: 20px;
+            top: 75px;
+            right: 20px;
             z-index: 1;
```

Comment:
I was able to position the zoom controls to the top-right. However, I noticed that resizing the page will cause the zoom plugins to move independently from the rest of our plugins.

![image](https://cloud.githubusercontent.com/assets/43062/19736575/e0511918-9b7d-11e6-9d46-04cbb946252a.png)

It "snaps" back into the correct position once you are done resizing the page. To avoid this, we might consider leaving the zoom controls in the top-left position.